glass-easel/src/backend/composed_backend_protocol.ts

```diff
@@ -30,21 +23,17 @@ export interface Context extends Partial<suggestedBackend.Context> {
   createFragment(): Element
   onEvent(
     listener: (
-      target: unknown,
+      target: GlassEaselElement,
```

Comment:
If shared a backend context with another glass-easel version or other frameworks, the event target may not be an `Element` of current glass-easel. Thus this should be unknown and requires an extra type check.

Reply:
> If shared a backend context with another glass-easel version or other frameworks, the event target may not be an `Element` of current glass-easel. Thus this should be unknown and requires an extra type check.

This type is to constraint backend's behavior, backends should pass a `glassEasel.Element` into this callback。

We can add an aditional type check inside this callback to solve the multi-version problem, but still need this type to tell backend developers that we are expecting a `glassEasel.Element` here.